index.js

```diff
@@ -23,6 +29,12 @@ async function main () {
 
     repos.forEach(repo => {
       console.log(repo.html_url)
+
+      completedRepoCount++
+      
+      if(completedRepoCount === repos.length){
+        spinnies.succeed('loading', { text: 'Done' })
```

Comment:
Just move this above line 21, before handling the error. Otherwise the spinner would continue to run even if an error occurs. Also this implementation (I haven't tested it) maybe prevents the console.logs since the spinner is still running up to this point

Reply:
Good spot, added an error failing message. 👍🏻